README.md

```diff
@@ -565,6 +573,46 @@ The configuration options for Graphite are:
 |`graphite.prefix`|No|e.g. `dev`|All metrics are prefixed with `prefix.namespace`. By default this is picked up from the `config` if provided.|
 | `graphite.namespace`|Yes|e.g. `com.company.team.circus-train.application`|All metrics are prefixed with `prefix.namespace`. Formatting of this is important for reporting from Graphite.|
 
+### Metadata transformations
+Circus Train can transform the metadata of a newly created replica table by adding table parameters during the replication.
+
+To use this feature, add properties to your configuration:
+
+        table-replications:
+           - ...
+        transform-options:
+          table-properties:
+            my-custom-property: my-custom-value
+            my-custom-property2: my-custom-value2
+
+This will add `my-custom-property` and `my-custom-property2` to the metadata of all tables in your replication. Properties can also be overridden for each replication:
+
+         table-replications:
+           - source-table:
+               ...
+             replica-table:
+               ...
+             transform-options:
+               table-properties:
+                 my-custom-override: my-custom-value
+           - source-table:
+               ...
+             replica-table:
+               ...
+         transform-options:
+           table-properties:
+            my-custom-param: my-custom-value
+            my-custom-param2: my-custom-value2
+
+In this case, the first table replication will add just one property, `my-custom-override`, while the second will add the two default properties. Adding any number of override properties to a replication will override all default properties. To add properties to your table with non-alphanumeric characters (other than -), surround the key with single quotes and brackets like so:
+
+        transform-options:
+          table-properties:
+            '[my.custom.property2]': my-custom-value
+            '[my.custom.property2]': my-custom-value2
+            
+This feature is a part of Circus Train and no further configuration is required. In addition to this, Circus Train supports custom metadata transformations which are built separately and placed on the Classpath when Circus Train is run. These are documented [below](#metadata-transformations-1).
```

Comment:
Maybe we should rename the section below to "Custom Metadata Transformations" (and update the anchor accordingly) as I think that's the main difference here. In this change you're adding a core on that anyone can use, while the section below describes how users can add their own.

Reply:
Agreed